streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/schedule/JobSchedulerImpl.java

```diff
@@ -67,8 +67,7 @@ public void schedule(JobGraph jobGraph, Map<String, String> jobConfig) {
           case PYTHON:
             byte[] workerContextBytes = buildPythonWorkerContext(
                 taskId, executionGraphPb, jobConfig);
-            waits.add(Ray.callPy((RayPyActor) worker,
-                "init", workerContextBytes));
+            waits.add(worker.call("init", workerContextBytes));
```

Comment:
I think we still need the cast: `(RayPyActor) worker`

Reply:
I added (RayPyActor) worker for calling Python actor method.